code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -180,6 +180,18 @@ update_label("John Doe", "Clowny")
 		var/datum/job/captain/J = new/datum/job/captain
 		access = J.get_access()
 		..()
+		
+/obj/item/weapon/card/id/lowpop_sec
+	name = "Acting Head of Security"
+	desc = "You're not just the last line of defense, you're the only line..."
+	icon_state = "silver"
+	item_state = "silver_id"
+	registered_name = "Acting Head of Security"

```

Comment:
`registered_name` is used for the owner of the card, this will cause their name to appear as "Acting Head of Security"